doc/dev/developer_guide/dash-devel.rst

```diff
@@ -1026,6 +1026,59 @@ Alerts and notifications
 Default notification should have `text-info` color. Success notification should
 have `text-success` color. Failure notification should have `text-danger` color.
 
+Accessibility
+.............
+
+We follow the `W3C WCAG2.1 <https://www.w3.org/TR/WCAG21/>`_  level A accessibility conformance guidelines. 
+By implementing accessibility best practices, you are also improving the usability of the site for all users.
+
+**Accessibility checkers:** During development, you can test the accessibility compliance of your features using one of the tools below
+
+- `Accessibility insights plugin <https://accessibilityinsights.io/downloads/>`_
+- `Site Improve plugin <https://www.siteimprove.com/integrations/browser-extensions/>`_
+- `Axe devtools <https://www.deque.com/axe/devtools/>`_
+
+Testing with two or more of these tools can greatly improve the detection of possible accessibility violations
+
+**Color contrast checkers:** When adding new colors, making sure they are accessible is also important, Here are some tools which can help with color contrast testing
```

Comment:
Suggest "When adding new colors, it is important to ensure that they are accessible.  Here are some tools that can help with contrast testing:"